recipes/xlntpyarrow/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xlntpyarrow" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "2c27a61a46ed84e1c20c5c581707eb95fe4c38fe19ea267400b7537017f9478f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: dev.zip
+  url: https://github.com/tfussell/xlnt/archive/dev.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win32]
+  skip: true  # [win and py<35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - cmake
+    - arrow
```

Comment:
Oops, I was wondering why there were two libraries called arrow. I'll fix that. Thanks @xhochy.